### PR TITLE
Fix media link

### PR DESCRIPTION
**Pull request by alavieille**
[OO-BUGFIX] Fix link with media in tinymce, replace tinymce plugin link by plugin orchestra_link

### CODE REVIEW

File: BackofficeBundle/Resources/config/form.yml [hunk 1]
```diff
@@ -54,10 +54,10 @@ parameters:
                 theme: modern
                 content_css: /built/openorchestrabackoffice/css/tinyMCEPatches/floatPanel.css
                 plugins:
-                    - advlist autolink lists link image charmap print preview hr anchor pagebreak
+                    - advlist autolink lists image charmap print preview hr anchor pagebreak
                     - searchreplace wordcount visualblocks visualchars code fullscreen
                     - insertdatetime media nonbreaking save table contextmenu directionality
-                    - emoticons template paste textcolor orchestra_bbcode
+                    - emoticons template paste textcolor orchestra_bbcode orchestra_link
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/config/form.yml]: Does the plugin's order declaration impact the rendering of the toolbar ? If so, you should replace the link with orchestra_link on the same location to make the modification transparent on a functionnal point of view

Comment (alavieille) [inline BackofficeBundle/Resources/config/form.yml]: no there isn't impact, the rendering of the toolbar is parameter by the key toolbar1

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**